### ISSUE TITLE
Training and Validation accuracy is constant

### ISSUE THREAD

**Issue by HussamCheema** [keras-team/keras#11791]
Why I'm getting constant training and validation accuracy in my model? What could be the reason? 
I have `(2364, 256, 256, 3)` shaped data of rgb images and `(2364, 8, 8)` shaped labels. 
labels categories are 1 to 7.

```
model = Sequential()
model.add(Conv2D(kernel_size=(3,3),filters=8,input_shape=(256, 256, 3),activation="relu",padding="valid"))
model.add(Conv2D(kernel_size=(3,3),filters=8,activation="relu",padding="same"))
model.add(MaxPooling2D(pool_size=(2,2),strides=(2,2)))
model.add(Flatten())
model.add(Dropout(0.6))
model.add(Dense(100,activation="relu"))
model.add(Dense(8,activation='softmax'))
model.summary()

model.compile(loss=keras.losses.categorical_crossentropy,
              optimizer=keras.optimizers.Adam(lr=0.001),
              metrics=['accuracy'])

X = model.fit(data, labels,
          batch_size=batch_size,
          epochs=epochs,
          verbose=1,
          shuffle=True,
          validation_split=0.2)

plt.plot(X.history['loss'])
plt.plot(X.history['val_loss'])
plt.title('model loss')
plt.ylabel('loss')
plt.xlabel('epoch')
plt.legend(['train', 'validation'], loc='upper left')
plt.show()
``` 
![image](https://user-images.githubusercontent.com/17239812/49449667-3bfca600-f7fd-11e8-8b13-2b3b4f47fb6a.png)

As you can see in this image, model is not learning at all. 

![image](https://user-images.githubusercontent.com/17239812/49449679-4323b400-f7fd-11e8-8304-dd3ed9ea1984.png)
